helix-term/src/commands/typed.rs

```diff
@@ -1656,6 +1656,13 @@ fn language(
         return Ok(());
     }
 
+    if args.len() == 0 {
+        let doc = doc!(cx.editor);
+        let language = &doc.language_name().unwrap_or("");
```

Comment:
```suggestion
        let language = &doc.language_name().unwrap_or("text");
```

Since we allow setting the language as `text` to clear the language config.

Also see here in the statusline element for the current buffer language: https://github.com/helix-editor/helix/blob/8a3ec443f176218384db8c8610bbada9c43a6ea5/helix-term/src/ui/statusline.rs#L401-L408

Reply:
I defined the constant `DEFAULT_LANGUAGE` in `helix-core/src/lib.rs`, maybe there is a better place? I figured language is not specific to `helix-term`.

Reply:
Ah nice, I like the re-use of the constant 👍

I think it might belong in `helix-view/src/document.rs` alongside the `DEFAULT_INDENT`